superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -143,62 +148,60 @@ const RETENTION_OPTIONS = [
   },
 ];
 
-const StyledModal = styled(Modal)`
-  max-width: 1200px;
-  width: 100%;
+const CONTENT_TYPE_OPTIONS = [
+  {
+    label: t('Dashboard'),
+    value: 'dashboard',
+  },
+  {
+    label: t('Chart'),
+    value: 'chart',
+  },
+];
+const FORMAT_OPTIONS = {
+  png: {
+    label: t('Send as PNG'),
+    value: 'PNG',
+  },
+  csv: {
+    label: t('Send as CSV'),
+    value: 'CSV',
+  },
+  txt: {
+    label: t('Send as text'),
+    value: 'TEXT',
+  },
+};
 
-  .ant-modal-body {
-    overflow: initial;
-  }
-`;
+// Style Constants
+const MODAL_BODY_HEIGHT = 180.5;
 
-const StyledTooltip = styled(InfoTooltipWithTrigger)`
-  margin-left: ${({ theme }) => theme.gridUnit}px;
-`;
+// Apply to collapse panels as 'style' prop value
+const panelBorder = {
+  borderBottom: 'none',
+};
 
-const StyledIcon = (theme: SupersetTheme) => css`
-  margin: auto ${theme.gridUnit * 2}px auto 0;
-  color: ${theme.colors.grayscale.base};
+// Apply to final text input components of each collapse panel
+const no_margin_bottom = css`
```

Comment:
We should use consistent naming conventions (though that's something that we fail to do in so many cases in Superset 😄). I'd suggest renaming it to `noMarginBottom`